op/model.py

```diff
@@ -314,3 +432,17 @@ def config_get(self):
 
     def is_leader(self):
         return self._run('is-leader')
+
+    def status_get(self, app):
```

Comment:
This looks like an app, but is a boolean. So we need a better name (is_app, for instance) and also need to force it to be provided as a kwarg so we prevent people from doing get(True) per above.

Reply:
Going to make it `is_app`.